.erda/schemas/dice.json

```diff
@@ -0,0 +1,742 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "https://github.com/erda-project/erda/.erda/schemas/dice.json",
```

Comment:
Can this be changed to dice.yaml.json here?
And pipeline.yaml.json too.

Reply:
done